src/app/(fullscreen)/detail/[id]/page.tsx

```
@@ -239,51 +239,45 @@ export default function Detail() {
             setModalType(null);
           }}
-          showCloseButton
-          onClose={handleModalClose}
-        >
-          {modalType === "loginRequired"
-            ? LOGIN_REQUIRED_MODAL.description
-            : currentModal?.description}
-        </Modal>
+        ></Modal>
+      ) : (
+        modalType && (
```

Biome (2.1.2)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)